rtsp.go

```diff
@@ -213,25 +235,23 @@ func (rc *rtspCamera) reconnectClient(codecInfo videoCodec) (err error) {
 }
 
 // initH264 initializes the H264 decoder and sets up the client to receive H264 packets.
-func (rc *rtspCamera) initH264(tracks media.Medias, baseURL *url.URL) (err error) {
-	var format *formats.H264
+func (rc *rtspCamera) initH264(session *description.Session) (err error) {
+	// setup RTP/H264 -> H264 decoder
+	var f *format.H264
+	var forma format.Format
 
-	track := tracks.FindFormat(&format)
-	if track == nil {
+	media := session.FindFormat(&f)
+	if media == nil {
 		rc.logger.Warn("tracks available")
-		for _, x := range tracks {
+		for _, x := range session.Medias {
 			rc.logger.Warnf("\t %v", x)
 		}
 		return errors.New("h264 track not found")
 	}
-
-	_, err = rc.client.Setup(track, baseURL, 0, 0)
-	if err != nil {
-		return err
-	}
+	forma = f
```

Comment:
why are we doing this instead of just using f?

Reply:
good call, removed